packages/node_modules/@webex/webex-core/src/config.js

```diff
@@ -19,7 +19,46 @@ export default {
   AlternateLogger: undefined,
   credentials: new CredentialsConfig(),
   services: {
-    u2c: process.env.U2C_SERVICE_URL || 'https://u2c.wbx2.com/u2c/api/v1'
+    /**
+     * A list of services that are available prior to catalog collection.
+     *
+     * @type {Object}
+     */
+    discovery: {
+      /**
+       * The hydra discovery url.
+       *
+       * @type {string}
+       */
+      hydra: process.env.HYDRA_SERVICE_URL || 'https://api.ciscospark.com/v1',
+
+      /**
+       * The u2c discovery url
+       *
+       * @type {string}
+       */
+      u2c: process.env.U2C_SERVICE_URL || 'https://u2c.wbx2.com/u2c/api/v1'
+    },
+
+    /**
+     * When true, enable whitelisted domain validation.
+     *
+     * @type {boolean}
+     */
+    validateDomains: process.env.VALIDATE_DOMAINS === 'true' || false,
+
+    /**
+     * Contains a list of whitelisted host addresses.
+     *
+     * @type {Array<string>}
+     */
+    whitelist: [
+      'webexkubed.com',
+      'wbx3.com',
+      'wbx2.com',
+      'ciscospark.com',
+      'ciscospark.com'
```

Comment:
This is a duplicate

Reply:
Currently, this will be a duplicate until we get to removing service methods from `internal-plugin-wdm`. In the mean time, we can do this if it helps any:

```js
const config = {
  ...
  services: {
    whitelist: [ ... ]
  }
  ...
};

config.device.whitelist = config.services.whitelist;
```

This saves memory, but increases diff.